server/lib/rooms/roomTypes/livechat.ts

```diff
@@ -22,16 +22,16 @@ roomCoordinator.add(LivechatRoomType, {
 		return ([RoomMemberActions.INVITE, RoomMemberActions.JOIN] as Array<ValueOf<typeof RoomMemberActions>>).includes(action);
 	},
 
-	roomName(room: any): string | undefined {
+	roomName(room: any, _userId?: string): string | undefined {
 		return room.name || room.fname || room.label;
 	},
 
 	canAccessUploadedFile({ rc_token: token, rc_rid: rid }): boolean {
 		return token && rid && LivechatRooms.findOneOpenByRoomIdAndVisitorToken(rid, token);
 	},
 
-	getNotificationDetails(room: IRoom, _user, notificationMessage: string) {
-		const title = `[Omnichannel] ${this.roomName(room)}`;
+	getNotificationDetails(room: IRoom, _sender, notificationMessage: string, userId?) {
```

Comment:
how did TS not screamed on the missing type?
```suggestion
	getNotificationDetails(room: IRoom, _sender: unknown, notificationMessage: string, userId?: string) {
```

Reply:
The function signature is already typed on the `IRoomTypeServerDirectives` interface.

Reply:
Wipe the all parameter types out to keep consistency.